config/crd/bases/bookkeeper.pravega.io_bookkeeperclusters.yaml

```diff
@@ -2825,6 +2825,10 @@ spec:
                       type: string
                   type: object
                 type: array
+              upgradeTimeout:
+                description: This is used to schedule the timeout value for rollback/upgrade
```

Comment:
suggestion --> `description: This is used to schedule the timeout value in minutes for rollback/upgrade`